### DISCUSSION TITLE
how to save and overwrite the results image?

### DISCUSSION THREAD

**Discussion by baktistr** [robmarkcole/yolov5-flask#38] [Q&A]
the new save() function seems to always make an incremental directory ( static/image0.jpg -> static2/image0.jpg -> static3/image0.jpg)

how to always overwrite 1 file, I tried it on docker ,,
but somehow when using python cli those phenomena doesn't ahappen

**Comment by robmarkcole** [comment]
@baktistr please see https://github.com/robmarkcole/yolov5-flask/pull/36
